lib/integrations/calendar/services/BaseCalendarService.ts

```diff
@@ -18,73 +18,62 @@ import { getLocation, getRichDescription } from "@lib/CalEventParser";
 import { symmetricDecrypt } from "@lib/crypto";
 import logger from "@lib/logger";
 
-import { CalendarEvent, IntegrationCalendar } from "./calendarClient";
+import { TIMEZONE_FORMAT } from "../constants/formats";
+import { CALDAV_CALENDAR_TYPE } from "../constants/generals";
+import { CalendarEventType, EventBusyDate, NewCalendarEventType } from "../constants/types";
+import { Calendar, CalendarEvent, IntegrationCalendar } from "../interfaces/Calendar";
+import { convertDate, getAttendees, getDuration } from "../utils/CalendarUtils";
 
-dayjs.extend(utc);
+const { CALENDSO_ENCRYPTION_KEY = "" } = process.env;
```

Comment:
We shouldn't destructure env variables according to [Next.js docs](https://nextjs.org/docs/api-reference/next.config.js/environment-variables):

> Next.js will replace process.env.customKey with 'my-value' at build time. Trying to destructure process.env variables won't work due to the nature of webpack DefinePlugin.

```suggestion
const CALENDSO_ENCRYPTION_KEY = process.env.CALENDSO_ENCRYPTION_KEY || "";
```

Reply:
Also it's a [security concern](https://nextjs.org/docs/basic-features/environment-variables): 

> Note: In order to keep server-only secrets safe, Next.js replaces process.env.* with the correct values at build time. This means that process.env is not a standard JavaScript object, so you’re not able to use object destructuring. Environment variables must be referenced as e.g. process.env.PUBLISHABLE_KEY, not const { PUBLISHABLE_KEY } = process.env.